versioned/tiered/tiered-tests/src/main/java/org/projectnessie/versioned/impl/SampleEntities.java

```diff
@@ -162,26 +162,26 @@ public static Id createId(Random random) {
     return Id.of(createBinary(random, 20));
   }
 
-  /**
-   * Create a String Entity of random characters.
-   * @param random random number generator to use.
-   * @param numChars the size of the String.
-   * @return the String Entity of random characters.
-   */
-  public static Entity createStringEntity(Random random, int numChars) {
-    return Entity.ofString(createString(random, numChars));
-  }
-
   /**
    * Create a String of random characters.
    * @param random random number generator to use.
    * @param numChars the size of the String.
    * @return the String of random characters.
    */
-  private static String createString(Random random, int numChars) {
+  public static String createString(Random random, int numChars) {
     return random.ints('a', 'z' + 1)
         .limit(numChars)
         .collect(StringBuilder::new, StringBuilder::appendCodePoint, StringBuilder::append)
         .toString();
   }
+
```

Comment:
Can you revert the move of createStringEntity?

Reply:
Done.